metecho/api/tests/models.py

```diff
@@ -777,6 +777,23 @@ def test_is_devhub_enabled__sf_error(self, user_factory, social_account_factory)
             get_devhub_api.return_value = client
             assert not user.is_devhub_enabled
 
+    def test_user_delete_triggers_scratch_org_delete(
+        self, user_factory, scratch_org_factory
+    ):
+        user = user_factory()
+        with ExitStack() as stack:
```

Comment:
`pytest-mock` provides a `mocker` fixture that replaces `ExitStack` (and the need for nesting in general). For new tests I would recommend we use [this approach](https://github.com/oddbird/Metecho/blob/658e2d5dceb1f3030c26d88e9a25e53a9adbea39/metecho/api/tests/admin.py#L101-L119) instead. You can find more examples by searching for `mocker.` in the tests.